draft-ietf-tls-dtls-rrc.md

```diff
@@ -160,6 +174,50 @@ threshold) and initiate the return routability check that proceeds as follows:
 After this point, any pending send operation is resumed to the bound peer
 address.
 
+{{path-challenge-reqs}} and {{path-response-reqs}} contain the requirments for
+the initiator and responder roles, broken down per protocol phase.
+
+## Path Challenge Requirements {#path-challenge-reqs}
+
+* The initiator MAY send multiple `return_routability_check` messages of type
+  path_challenge to cater for packet loss on the probed path.
+  * Each path_challenge SHOULD go into different transport packets.
```

Comment:
make this a MUST and say that the receiver MUST NOT enforce this (because the sender may have no control over the repackaging made at the transport layer)

Reply:
On further reflection, I don't think we can impose a MUST that is outside the control of the DTLS stack.

Reply:
added a note in df82fd8